packages/app-studio-toolkit/src/utils/bas-utils.ts

```diff
@@ -58,3 +64,34 @@ function getExtensionRunPlatform(): ExtensionRunMode {
 
   return runPlatform;
 }
+
+export async function reportProjectTypesToUsageAnalytics(
+  basToolkitAPI: BasToolkit
+) {
+  const devspaceInfo = await devspace.getDevspaceInfo();
+  const projects = await getProjectsInfo(basToolkitAPI);
```

Comment:
these two calls are independent of each other, consider calling at the same time, e.g. Promise.all (...)